components/contribution-flow/index.js

```diff
@@ -622,7 +623,7 @@ class CreateOrderPage extends React.Component {
         name: 'details',
         isCompleted: Boolean(stepDetails && stepDetails.totalAmount >= minAmount),
         validate: () => {
-          return stepDetails && this.activeFormRef.current && this.activeFormRef.current.reportValidity();
+          return stepDetails && this.activeFormRef.current && reportValidityHTML5(this.activeFormRef.current);
```

Comment:
Thanks to the new helper, we can simplify this line

```suggestion
          return stepDetails && reportValidityHTML5(this.activeFormRef.current);
```

Reply:
Thanks :)